### DISCUSSION TITLE
How to upgrade Postal Server

### DISCUSSION THREAD

**Discussion by impirecom** [postalserver/postal#2557] [Help with using Postal]
Hello,

I looked at the discussion on auto upgrade, but cannot find the section as mentioned "Postal Server", "Auto Upgrade".

Could someone please share the steps to setup auto upgrade, or even how to manual upgrade? Thank you much.

Kind regards,

Andy

**Comment by willpower232** [comment]
That sounds like a very old discussion from before postal ran on docker, have a look at https://docs.postalserver.io/install/upgrading

**Reply by impirecom** [reply]
Thank you @willpower232. Do you know how I can get paid support for Postal Server? I don't see an option to purchase this service.

**Reply by willpower232** [reply]
I am not aware of it being offered officially but there have been people asking for paid support elsewhere in these issues and discussions so would be worth having a look and see how they got on.

**Comment by alex-pommier** [comment]
Hello @impirecom 
I posted a similar thread asking how to manually upgrade if it's helpful to you.
https://github.com/postalserver/postal/discussions/2488
